CHANGELOG.md

```diff
@@ -44,6 +44,8 @@ Changelog
 * RADIUS
   * radius_global (@jonnytpuppet)
   * radius_server (@jonnytpuppet)
+*SNMP
```

Comment:
missing space between `*` and `SNMP` is causing the markdown to render this as italics instead of a bullet list - please fix.